source/auth/auth.rst

```diff
@@ -863,18 +860,27 @@ Body                     Action=GetCallerIdentity&Version=2011-06-15\\n
         ``*``, Denotes a header that MUST be included in SignedHeaders, if present.
 
 .. note::
```

Comment:
`.. note::` is not rendered correctly by Github. Any chance you can use something else? Maybe a new section:
```
Region                   Derived from Host - see `Region Calculation`_ below.
...

Region Calculation
~~~~~~~~~~~~~~~~~~

``Region`` is not a header, ...
```

Or make this a new section:

Reply:
Done